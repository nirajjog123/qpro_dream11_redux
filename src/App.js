import React from "react";
import "./App.css";
import { connect } from "react-redux";
import { BrowserRouter as Router, Route, Link } from "react-router-dom";
import CreateTeam from "./createTeam/createTeam";
import ViewTeam from "./viewTeam/viewTeam";
import detailTeam from "./detailTeam/detailTeam";
import listTeam from "./listTeam/listTeam";

function App() {
  return (
    <div className="App">
      <Router>
        <div className="topnav">
          <img src="./images/logo.png" className="handleLogo" alt="logo" />
          <Link to="/createteam">Create Team</Link>
          <Link to="/viewteam">View Teams</Link>
        </div>
        <Route  path="/createteam" component={CreateTeam} />
        <Route path="/viewteam" component={ViewTeam} />
        <Route path="/detailteam" component={detailTeam} />
        <Route exact path="/" component={listTeam}/>
      </Router>
    </div>
  );
}

export default connect(
  null,
  null
)(App);
