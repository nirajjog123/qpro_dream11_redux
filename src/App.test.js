import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import 'jest-dom/extend-expect';
import { shallow, configure , mount} from 'enzyme';
import { BrowserRouter as Router } from "react-router-dom";
import App from './App';

configure({adapter: new Adapter()});

 describe('<App />', () => {
   const { container } = shallow(<App />);
  it('it should render properly ', () => {
   expect(container).toMatchSnapshot();
  })

//   it('valid path should not redirect to 404', () => {
//     const wrapper = shallow(
//       <Router initialEntries={[ '/' ]}>
//         <App/>
//       </Router>
//     )
//     expect(wrapper.find(App)).toHaveLength(0);
//   })
 });
