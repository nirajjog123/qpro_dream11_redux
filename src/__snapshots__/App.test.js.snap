// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> it should render properly  1`] = `undefined`;
